_layouts/lesson.html

```diff
@@ -183,25 +183,18 @@ <h4>{{site.data.snippets.difficulty[page.lang]}}</h4> {% include difficulty.html
   {% include support-alert.html %}
   {% endif %}
 
-  {% unless page == translation_source %}
-  <!-- If this lesson is translated, banner pointing to the original language source of this lesson -->
-  <div class="alert alert-warning">
-    {{ site.data.snippets.translation-source[page.lang] }} {{ site.data.snippets.language-name.en[page.lang]}}: <a
-      href="{{ translation_source.url }}">{{ translation_source.title }}</a>
-  </div>
-  {% endunless %}
 
-  {% for candidate in translation_candidates %}
-  {% unless page.lang == candidate.lang %}
-  <!-- Banner pointing to translations of this lesson when they exist -->
-  <div class="alert alert-warning">
-    {{ site.data.snippets.translation-target[page.lang]}}
-    {{ site.data.snippets.language-name[candidate.lang][page.lang]}}: <a
-      href="{{ candidate.url }}">{{ candidate.title }}</a>
+ {% for candidate in translation_candidates %} 
```

Comment:
Think about the logic of what you are trying to accomplish: you want a single `<div>` banner, and then loop through available lessons within it. So the `for` loop needs to start _inside_ the div, and _after_ the part of the alert that says "available in" / "Disponible en" etc. Move the for loop so that it's just looping after the colon `:`.

Use `/fr/lecons/analyse-corpus-antconc` as a test lesson since it will show you immediately where things are going wrong.